app/presenters/webauthn_setup_presenter.rb

```diff
@@ -20,6 +20,7 @@ def initialize(
       remember_device_default: remember_device_default,
     )
 
+    @user = current_user
```

Comment:
we pass `current_user` to `super()` and it is accessible via `attr_reader :current_user`, can we remove this and rely on that?